topics/epigenetics/tutorials/hicexplorer/tutorial.md

```diff
@@ -0,0 +1,299 @@
+>    - "Include chromosomes" to `chr1 chr2 chr3 chr4 chr5 chr6 chr7 chr8 chr9 chr10 chr11 chr12 chr13 chr14 chr15 chr16 chr17 chr18 chr19 chrX chrY`
+>
+{: .hands_on}
+
+It can happen that the correction stops with:
+
+```
+ERROR:iterative correction:*Error* matrix correction produced extremely large values.
+This is often caused by bins of low counts. Use a more stringent filtering of bins.
+```
+
+This can be solved by a more stringent z-score values for the filter threshold or by a look at the plotted matrix. In our case we see that chromosome Y is having more or less 0 counts in its bins. This chromosome can be excluded from the correction by not defining it for the set of chromosomes that should be corrected (parameter 'Include chromosomes').
+
+### Plotting the corrected Hi-C matrix
+
+We can now plot the one of the chromosomes (e.g. chromosome X) , with the corrected matrix.
+
+> ### {% icon hands_on %} Hands-on: Plotting the corrected Hi-C matrix
+>
+> 1. **hicPlotMatrix** {% icon tool %}: Run hicPlotMatrix on the corrected matrix adjusting the parameters:
+>    - "Plot title" to `Hi-C matrix for mESC`
+>    - "Plot per chromosome" to `False`
+>    - "Chromosomes to include (and order to plot in)" to `chr1 chr2 chr3 chr4 chr5 chr6 chr7 chr8 chr9 chr10 chr11 chr12 chr13 chr14 chr15 chr16 chr17 chr18 chr19 chrX chrY`
+>    - "Plot only this region" to `chrX` 'or' `X`
+>    - "Plot the log1p of the matrix values" to `True`
+>
+{: .hands_on}
+
+# TAD calling
+
+“The partitioning of chromosomes into topologically associating domains (TADs) is an emerging concept that is reshaping our understanding of gene regulation in the context of physical organization of the genome” [Ramirez et al. 2017](https://doi.org/10.1101/115063).
+
+TAD calling works in two steps: First HiCExplorer computes a TAD-separation score based on a z-score matrix for all bins. Then those bins having a local minimum of the TAD-separation score are evaluated with respect to the surrounding bins to decide assign a p-value. Then a cutoff is applied to select the bins more likely to be TAD boundaries.
+
+[hicFindTADs](http://hicexplorer.readthedocs.io/en/latest/content/tools/hicFindTADs.html#hicfindtads) tries to identify sensible parameters but those can be change to identify more stringent set of boundaries.
+
+> ### {% icon hands_on %} Hands-on: Finding TADs
+>
+> 1. **hicFindTADs** {% icon tool %}: Run hicFindTADs on the corrected matrix adjusting the parameters:
+>    - "Minimum window length (in bp) to be considered to the left and to the right of each Hi-C bin." to `60000`
+>    - "Maximum window length (in bp) to be considered to the left and to the right of each Hi-C bin." to `120000`
+>    - "Step size when moving from minDepth to maxDepth" to `20000`
+>    - "Multiple Testing Corrections" to `False discovery rate`
+>    - "q-value" to `0.05` 
+>    - "Minimum distance between boundaries (in bp)." to `80000`
+>
+{: .hands_on}

```

Comment:
New values are:
min: 30000
max: 60000
step: 10000
fdr: 0.05
threshold: 0.05

remove `"Minimum distance between boundaries (in bp)." to 80000`